internal/app/query/query_service.go

```diff
@@ -42,8 +41,9 @@ var (
 // queryService API provides utility functions for common MMF functionality such
 // as retreiving Tickets from state storage.
 type queryService struct {
-	cfg config.View
-	tc  *ticketCache
+	cfg     config.View
+	tc      *ticketCache
```

Comment:
probably rename to ticketCache?

Reply:
or `tCache`. We don't want to change that, because we didn't want to change Ticket core logic and produce some bugs. So let's stay as is.

Reply:
It is used just in 3 places and it won't cause bugs because it won't compile if something goes wrong.
Just noticed a different naming style for backfill and ticket cache.

Reply:
I standarized the field naming as it always have been